src/controllers/TerminalController.java

```diff
@@ -95,9 +94,13 @@ private String processSpelling(String inputtedCommand) {
 
     private Integer processNumber(String inputtedCommand) {
         if (NumberUtils.isParsable(inputtedCommand)){
```

Comment:
If statement un-necessary given the try-catch block. The return null in the catch block is redundant if the first if statement is removed.